runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -110,6 +110,10 @@ public Dimension render(Graphics2D graphics)
 
 			if (!plugin.isHotKeyPressed())
 			{
+				if (plugin.isHideAll())
```

Comment:
This should be at beginning of the method. Checks here are related to individual items, your boolean is related to all.